recipes/box2d/2.4.x/conanfile.py

```diff
@@ -1,5 +1,8 @@
 import os
-from conans import ConanFile, CMake, tools
+from conan import ConanFile
+from conan.tools.files import apply_conandata_patches, get, rm, rmdir
+from conan.tools.cmake import CMake, CMakeToolchain
+from conan.tools.scm import Version
 
 required_conan_version = ">=1.29.1"
```

Comment:
```suggestion
required_conan_version = ">=1.52.0"
```

Asking for 1.52.0 because export_conandata_patches is a good addition now.